src/atwho.js

```diff
@@ -109,7 +109,7 @@ define([
         });
 
         if (options.useRichText) {
-            options.insertTpl = '${name}';
+            options.insertTpl = '${hashtagPath}';
```

Comment:
I didn't understand that, so I'm not sure what to take away. Is there some cleanup that could be done make this less messy?